spec/system/candidate_interface/submitting/candidate_submitting_application_spec.rb

```diff
@@ -212,7 +212,9 @@ def then_i_can_see_my_application_dashboard
   end
 
   def when_i_click_view_application
-    click_link 'View application'
+    within '.govuk-hint' do
+      click_link 'View application'
+    end
```

Comment:
Is this really within hint text somewhere?

Reply:
Yep. See https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/views/candidate_interface/submitted_application_form/complete.html.erb#L9

Let me know if you want changes here @frankieroberto 👍

Reply:
@TobyRet ah yes! That will get changed with #4362, so I think you can leave this as is for now.